content/docs/esc/integrations/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Integrations
+title_tag: Pulumi ESC integrations and providers
+meta_desc: Explore Pulumi ESC integrations and providers that help you securely manage cloud resources, configurations, and secrets.
+h1: Pulumi ESC integrations
+menu:
+    pulumiesc:
+        weight: 8
+        identifier: esc-integrations
+aliases:
+    - /docs/esc/providers/
+---
+
+Pulumi ESC support for dynamic configuration providers allow ESC to integrate with secrets stored in any other provider. Organizations often use AWS OIDC, AWS Secrets Manager, Vault, Azure OIDC, Azure KeyVault, GCP OIDC, and GCP Secrets Manager plus many more sources of truth for their secrets and configuration. Pulumi ESC supports them all, providing a single interface to your configuration and secrets, no matter where their source of truth is. Pulumi ESC works with these tools to provide improved management of secrets and configuration.
+
+ESC also integrates with tools like Direnv, Terraform, and Docker to help manage environment variables, interact with existing infrastructure, and support containerized workflows.
+
+## Dynamic login providers
+
+- [AWS login provider](/docs/esc/integrations/dynamic-login-credentials/aws-login)
+- [Azure login provider](/docs/esc/integrations/dynamic-login-credentials/azure-login)
+- [GCP login provider](/docs/esc/integrations/dynamic-login-credentials/gcp-login)
+- [Vault login provider](/docs/esc/integrations/dynamic-login-credentials/vault-login)
+
+## Dynamic secrets providers
```

Comment:
As an editorial note, I feel like we're leaning too heavily on the word "dynamic" in a lot of this stuff, and while I get it, it does feel a bit vague and repetitive. The key thing we want people to see is that there are two kinds of integration -- one for obtaining provider credentials specifically, the other for obtaining third-party secrets more generally. The "dynamic"-ness of both is sort of a given (i.e., in that we "fetch stuff") and seems superfluous, considering there isn't really a "static" version of either one, if that makes sense.

Reply:
Understood, let's assess this into next week.